.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 * @epoberezkin @efim-poberezkin
-/*Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
+/Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
```

Comment:
There isn't a single `Dockerfile` in the root of this repository currently.

Reply:
Will be after this merged: https://github.com/simplex-chat/simplexmq/pull/750